modules/mobile/app/services/mobile/v0/vaccinations/service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
```

Comment:
If we added manufacturer info, it could also be added here by pulling it from the other xml file.